.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Parsers/DutchHolidayParserConfiguration.cs

```diff
@@ -1,100 +1,101 @@
-            {
-                swift = -1;
-            }
-            else if (trimmedText.StartsWith("deze"))
-            {
-                swift = 0;
-            }
-
-            return swift;
-        }
-
-        public override string SanitizeHolidayToken(string holiday)
-        {
-            return holiday
-                .Replace(" ", string.Empty)
-                .Replace("'", string.Empty);
-        }
-
-        protected override IDictionary<string, Func<int, DateObject>> InitHolidayFuncs()
-        {
-            return new Dictionary<string, Func<int, DateObject>>(base.InitHolidayFuncs())
-            {
-                { "maosbirthday", MaoBirthday },
-                { "yuandan", NewYear },
-                { "teachersday", DutchTeachersDay },
-                { "singleday", SinglesDay },
-                { "allsaintsday", HalloweenDay },
-                { "youthday", YouthDay },
-                { "childrenday", ChildrenDay },
-                { "femaleday", FemaleDay },
-                { "treeplantingday", TreePlantDay },
-                { "arborday", TreePlantDay },
-                { "girlsday", GirlsDay },
-                { "whiteloverday", WhiteLoverDay },
-                { "loverday", ValentinesDay },
-                { "christmas", ChristmasDay },
-                { "xmas", ChristmasDay },
-                { "newyear", NewYear },
-                { "newyearday", NewYear },
-                { "newyearsday", NewYear },
-                { "inaugurationday", InaugurationDay },
-                { "groundhougday", GroundhogDay },
-                { "valentinesday", ValentinesDay },
-                { "stpatrickday", StPatrickDay },
-                { "aprilfools", FoolDay },
-                { "stgeorgeday", StGeorgeDay },
-                { "mayday", Mayday },
-                { "cincodemayoday", CincoDeMayoday },
-                { "baptisteday", BaptisteDay },
-                { "usindependenceday", UsaIndependenceDay },
-                { "independenceday", UsaIndependenceDay },
-                { "bastilleday", BastilleDay },
-                { "halloweenday", HalloweenDay },
-                { "allhallowday", AllHallowDay },
-                { "allsoulsday", AllSoulsday },
-                { "guyfawkesday", GuyFawkesDay },
-                { "veteransday", DutchVeteransday },
-                { "christmaseve", ChristmasEve },
-                { "newyeareve", NewYearEve },
-                { "easterday", EasterDay },
-                { "kingsday", KingsDay },
-                { "queensday", QueensDay },
-                { "prinsjesdag", Prinsjesdag },
-                { "eerstekerstdag", ChristmasDay },
-                { "tweedekerstdag", SecondChristmasDay },
-                { "bevrijdingsdag", Bevrijdingsdag },
-                { "dodenherdenking", Dodenherdenking },
-                { "dagvandearbeid", Dagvandearbeid },
-            };
-        }
-
+﻿using System;

```

Comment:
Are these line-break changes really necessary to be consistent with other files?
If not, please revert and be careful not to make such changes unless really necessary.

Reply:
the line ending on this file was Unix style which is inconsistent with other similar files. 
if that is not an issue we can revert this.

Reply:
If this is now consistent with the rest of the codebase that's fine.

Reply:
If such cases happen in the future, please make a comment in the PR description to clarify. Thanks!